tox.ini

```diff
@@ -158,4 +158,4 @@ commands = python -c 'import plasmapy'
 deps =
     mypy >= 1.8.0
```

Comment:
I'm wondering if there's an alternative to having to update this manually...  dependabot, maybe?

Reply:
:thinking: